requirements/dev.txt

```diff
@@ -0,0 +1,11 @@
+black[jupyter]
+flake8
+isort
+prophet
```

Comment:
Do we need Prophet here? Was only added to NP due to some migration samples and tests